sources/mimeogram/acquirers.py

```
@@ -56,15 +57,23 @@ async def acquire(
                 tasks.append( _produce_http_task( str( source ) ) )
             case _:
                 raise _exceptions.UrlSchemeNoSupport( str( source ) )
-    if strict: return await __.gather_async( *tasks )
-    results = await __.gather_async( *tasks, return_exceptions = True )
+    if strict:
+        return await __.gather_async( *tasks )
+    # Use asyncio.gather for non-strict mode to ensure coroutines are awaited
+    results = await asyncio.gather( *tasks, return_exceptions = True )
```

Do you know what GPT 4.1's problem with the existing code was? As far as I know, `__.gather_async` should properly await the coroutines and handle unwrapping of valid results automatically.

---

Well, I reverted the change and the tests are all passing. I'll go through the rest of the review items, retest, recompile and check if it works still.

I was manually installing dependencies with pip because I didn't know about hatch (very ignorant of python). I ran into a bunch of annoying silent failures because of the absence of patiencediff, it just quietly didn't make the diff or something IIRC. Anyway, this change may have come out of when I was struggling with that.

Let me retest everything without it.

---

Alright, all tests are passing, built the executable and used it to perfectly regenerate my existing mimeograms. So didn't need this change, sorry about that.